shared/chat/conversation/messages/reactions-row/container.js

```diff
@@ -4,6 +4,19 @@ import * as Constants from '../../../../constants/chat2'
 import * as Types from '../../../../constants/types/chat2'
 import ReactionsRow from '.'
 
+// Get array of emoji names in the order of their earliest reaction
+const getOrderedReactions = (reactions: ?Types.Reactions) => {
+  if (!reactions) {
+    return []
+  }
+  const mins = reactions
```

Comment:
🐑  you can just do (reactions || []) and remove that above if

Reply:
`reactions` is a `Map` so I'd need to add an import to do that